src/examples/cicd-s3-client.yml

```diff
@@ -0,0 +1,39 @@
+description: |
+  This example shows how a frontend application being deployed to S3 and CF
+  can be setup with a CICD pipeline using this orb.
+
+usage:
+  version: 2.1
+
+  orbs:
+    cyber4all: cyber4all/orb:1.0.0
+
+  workflows:
+    integration-testing:
+      jobs:
+        - cyber4all/lint-app:
+            docker-image: cimg/node
+            isNode: true
+
+        - cyber4all/deploy-s3:
+            deploy: false # explicitly shows in develeopment
+            filters:
+              branches:
+                ignore: releases
+
+        - cyber4all/test-app:
+            docker-image: cimg/node
+            node-test: test:lighthouse
```

Comment:
CLARK client is the only client we have lighthouse tests on. We do not have true testing on any of our clients for that matter. This will need to be revised in the future, but in the mean time, we should just run 'npm test'

Reply:
Ok, that is the default, so this is meant to be a 'template'